app/Controllers/LibriController.php

```
@@ -354,7 +354,7 @@ public function show(Request $request, Response $response, mysqli $db, int $id):
         }
         $loanRepo = new \App\Models\LoanRepository($db);
         $activeLoan = $loanRepo->getActiveLoanByBook($id);
-        $bookPath = '/admin/libri/' . $id;
+        $bookPath = url('/admin/libri/' . $id);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$bookPath'. (undefined)

(UnusedLocalVariable)